scripts/ev_code_signing.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Not used on a day to day basis, but I kept it here so that it is easy to test code signing without the full build system.